recipes/higra/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "higra" %}
+{% set version = "0.6.8" %}
+{% set sha256 = "4b86b31bac16c216048f29c5ddd7660a0712770d1de055f9825514fa4d64a2c6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/higra/Higra/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
Please don't add this indirection, add the `sha256` directly here.

Reply:
Done!